assets/src/settings-page/sandboxing.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { useContext } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { CheckboxControl } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { Options } from '../components/options-context-provider';
+import { AMPDrawer } from '../components/amp-drawer';
+import { STANDARD } from '../common/constants';
+
+/**
+ * Component rendering the Sandboxing experiment.
+ *
+ * @param {Object} props                Component props.
+ * @param {string} props.focusedSection Focused section.
+ */
+export function Sandboxing( { focusedSection } ) {
+	const { updateOptions, editedOptions: {
+		sandboxing_enabled: sandboxingEnabled,
+		sandboxing_level: sandboxingLevel,
+		theme_support: themeSupport,
+	} } = useContext( Options );
+
+	if ( STANDARD !== themeSupport ) {
```

Comment:
Since this is an experimental feature, should it be available to technical users only?

Reply:
That's a good question. I think that it should be available to non-technical users too, as this feature is intended to be the most benefit to users who are not technical in the first place, since it avoids the need to fix validation errors.